Mathlib/LinearAlgebra/FreeModule/Finite/Basic.lean

```diff
@@ -49,3 +50,14 @@ instance Module.Finite.matrix {R ι₁ ι₂ M : Type*}
 
 example {ι₁ ι₂ R : Type*} [Semiring R] [Finite ι₁] [Finite ι₂] :
     Module.Finite R (Matrix ι₁ ι₂ R) := inferInstance
+
+variable {R M : Type*} [CommRing R] [AddCommGroup M] [Module R M] [IsDomain R]
+  [IsPrincipalIdealRing R] [Module.Finite R M] [NoZeroSMulDivisors R M] [Module.Free R M]
+
+variable (R M) in
+/-- The isomorphism `M ≃ₗ[R] (Fin (Module.finrank R M) → R)`. -/
+noncomputable def Module.Free.finRepr : M ≃ₗ[R] (Fin (Module.finrank R M) → R) :=
```

Comment:
Is there a shorter path via `Basis (Fin <| Module.finrank R M) R M`?

Reply:
Actually, `(Module.finBasis _ _).repr` is short enough for me that I don't need this PR!

Reply:
Either we should add this definition, or deprecate `Module.free.repr`; both are equally trivial.

Reply:
Do you have a preference?

Reply:
No; but if we decide to have both, they should probably both be `abbrev`s